<?xml version="1.0" encoding="UTF-8"?>
<interface>
  <!-- interface-requires gtk+ 3.0 -->
  <object class="GtkImage" id="add_image">
    <property name="visible">True</property>
    <property name="can_focus">False</property>
    <property name="stock">gtk-add</property>
  </object>
  <object class="GtkTextBuffer" id="definitions_textbuffer">
    <signal name="paste-done" handler="gw_addvocabularywindow_definitions_paste_done_cb" object="toplevel" swapped="no"/>
  </object>
  <object class="GtkDialog" id="unused">
    <property name="can_focus">False</property>
    <property name="border_width">5</property>
    <property name="type_hint">dialog</property>
    <child internal-child="vbox">
      <object class="GtkBox" id="toplevel">
        <property name="can_focus">False</property>
        <property name="margin_left">4</property>
        <property name="margin_right">4</property>
        <property name="margin_bottom">4</property>
        <property name="orientation">vertical</property>
        <property name="spacing">2</property>
        <child internal-child="action_area">
          <object class="GtkButtonBox" id="dialog-action_area1">
            <property name="can_focus">False</property>
            <property name="layout_style">end</property>
            <child>
              <object class="GtkButton" id="cancel_button">
                <property name="label">gtk-cancel</property>
                <property name="use_action_appearance">False</property>
                <property name="visible">True</property>
                <property name="can_focus">True</property>
                <property name="receives_default">True</property>
                <property name="use_action_appearance">False</property>
                <property name="use_stock">True</property>
                <signal name="activate" handler="gw_addvocabularywindow_cancel_cb" object="toplevel" swapped="no"/>
                <signal name="clicked" handler="gw_addvocabularywindow_cancel_cb" object="toplevel" swapped="no"/>
              </object>
              <packing>
                <property name="expand">False</property>
                <property name="fill">True</property>
                <property name="position">0</property>
              </packing>
            </child>
            <child>
              <object class="GtkButton" id="add_button">
                <property name="label" translatable="yes">_Add Vocabulary Word</property>
                <property name="use_action_appearance">False</property>
                <property name="visible">True</property>
                <property name="can_focus">True</property>
                <property name="can_default">True</property>
                <property name="has_default">True</property>
                <property name="receives_default">True</property>
                <property name="use_action_appearance">False</property>
                <property name="image">add_image</property>
                <property name="use_underline">True</property>
                <signal name="activate" handler="gw_addvocabularywindow_add_cb" object="toplevel" swapped="no"/>
                <signal name="clicked" handler="gw_addvocabularywindow_add_cb" object="toplevel" swapped="no"/>
              </object>
              <packing>
                <property name="expand">False</property>
                <property name="fill">True</property>
                <property name="position">1</property>
              </packing>
            </child>
          </object>
          <packing>
            <property name="expand">False</property>
            <property name="fill">True</property>
            <property name="pack_type">end</property>
            <property name="position">0</property>
          </packing>
        </child>
        <child>
          <object class="GtkBox" id="content_box">
            <property name="visible">True</property>
            <property name="can_focus">False</property>
            <child>
              <object class="GtkBox" id="box1">
                <property name="visible">True</property>
                <property name="can_focus">False</property>
                <property name="orientation">vertical</property>
                <child>
                  <object class="GtkImage" id="add_big_image">
                    <property name="visible">True</property>
                    <property name="can_focus">False</property>
                    <property name="margin_left">8</property>
                    <property name="margin_right">8</property>
                    <property name="margin_top">8</property>
                    <property name="margin_bottom">8</property>
                    <property name="stock">gtk-add</property>
                    <property name="icon-size">6</property>
                  </object>
                  <packing>
                    <property name="expand">False</property>
                    <property name="fill">True</property>
                    <property name="position">0</property>
                  </packing>
                </child>
              </object>
              <packing>
                <property name="expand">False</property>
                <property name="fill">True</property>
                <property name="padding">12</property>
                <property name="position">0</property>
              </packing>
            </child>
            <child>
              <object class="GtkBox" id="data_box">
                <property name="visible">True</property>
                <property name="can_focus">False</property>
                <property name="border_width">6</property>
                <property name="orientation">vertical</property>
                <property name="spacing">5</property>
                <child>
                  <object class="GtkBox" id="list_box">
                    <property name="visible">True</property>
                    <property name="can_focus">False</property>
                    <property name="spacing">8</property>
                    <child>
                      <object class="GtkLabel" id="list_label">
                        <property name="visible">True</property>
                        <property name="can_focus">False</property>
                        <property name="xalign">0</property>
                        <property name="label" translatable="yes">Vocabulary List:</property>
                      </object>
                      <packing>
                        <property name="expand">False</property>
                        <property name="fill">True</property>
                        <property name="position">0</property>
                      </packing>
                    </child>
                    <child>
                      <object class="GtkComboBox" id="vocabulary_list_combobox">
                        <property name="visible">True</property>
                        <property name="can_focus">False</property>
                        <property name="has_entry">True</property>
                        <child internal-child="entry">
                          <object class="GtkEntry" id="combobox-entry2">
                            <property name="can_focus">False</property>
                            <signal name="changed" handler="gw_addvocabularywindow_list_changed_cb" object="toplevel" swapped="no"/>
                          </object>
                        </child>
                      </object>
                      <packing>
                        <property name="expand">True</property>
                        <property name="fill">True</property>
                        <property name="position">1</property>
                      </packing>
                    </child>
                  </object>
                  <packing>
                    <property name="expand">False</property>
                    <property name="fill">False</property>
                    <property name="position">0</property>
                  </packing>
                </child>
                <child>
                  <object class="GtkBox" id="kanji_box">
                    <property name="visible">True</property>
                    <property name="can_focus">False</property>
                    <property name="spacing">8</property>
                    <child>
                      <object class="GtkLabel" id="kanji_label">
                        <property name="visible">True</property>
                        <property name="can_focus">False</property>
                        <property name="xalign">0</property>
                        <property name="label" translatable="yes">Kanji:</property>
                      </object>
                      <packing>
                        <property name="expand">False</property>
                        <property name="fill">True</property>
                        <property name="position">0</property>
                      </packing>
                    </child>
                    <child>
                      <object class="GtkEntry" id="kanji_entry">
                        <property name="visible">True</property>
                        <property name="can_focus">True</property>
                        <property name="has_focus">True</property>
                        <property name="invisible_char">‚óè</property>
                        <property name="activates_default">True</property>
                        <property name="caps_lock_warning">False</property>
                        <signal name="changed" handler="gw_addvocabularywindow_kanji_changed_cb" object="toplevel" swapped="no"/>
                      </object>
                      <packing>
                        <property name="expand">True</property>
                        <property name="fill">True</property>
                        <property name="position">1</property>
                      </packing>
                    </child>
                  </object>
                  <packing>
                    <property name="expand">False</property>
                    <property name="fill">False</property>
                    <property name="position">1</property>
                  </packing>
                </child>
                <child>
                  <object class="GtkBox" id="furigana_box">
                    <property name="visible">True</property>
                    <property name="can_focus">False</property>
                    <property name="spacing">8</property>
                    <child>
                      <object class="GtkLabel" id="furigana_label">
                        <property name="visible">True</property>
                        <property name="can_focus">False</property>
                        <property name="xalign">0</property>
                        <property name="label" translatable="yes">Furigana:</property>
                      </object>
                      <packing>
                        <property name="expand">False</property>
                        <property name="fill">True</property>
                        <property name="position">0</property>
                      </packing>
                    </child>
                    <child>
                      <object class="GtkEntry" id="furigana_entry">
                        <property name="visible">True</property>
                        <property name="can_focus">True</property>
                        <property name="invisible_char">‚óè</property>
                        <property name="activates_default">True</property>
                        <property name="caps_lock_warning">False</property>
                        <signal name="changed" handler="gw_addvocabularywindow_furigana_changed_cb" object="toplevel" swapped="no"/>
                      </object>
                      <packing>
                        <property name="expand">True</property>
                        <property name="fill">True</property>
                        <property name="position">1</property>
                      </packing>
                    </child>
                  </object>
                  <packing>
                    <property name="expand">False</property>
                    <property name="fill">False</property>
                    <property name="position">2</property>
                  </packing>
                </child>
                <child>
                  <object class="GtkBox" id="definitions_box">
                    <property name="visible">True</property>
                    <property name="can_focus">False</property>
                    <property name="spacing">8</property>
                    <child>
                      <object class="GtkLabel" id="definitions_label">
                        <property name="visible">True</property>
                        <property name="can_focus">False</property>
                        <property name="xalign">0</property>
                        <property name="label" translatable="yes">Definitions:</property>
                      </object>
                      <packing>
                        <property name="expand">False</property>
                        <property name="fill">True</property>
                        <property name="position">0</property>
                      </packing>
                    </child>
                    <child>
                      <object class="GtkScrolledWindow" id="definitions_scrolledwindow">
                        <property name="visible">True</property>
                        <property name="can_focus">True</property>
                        <property name="hscrollbar_policy">never</property>
                        <property name="vscrollbar_policy">never</property>
                        <property name="shadow_type">in</property>
                        <property name="min_content_height">70</property>
                        <child>
                          <object class="GtkTextView" id="definitions_textview">
                            <property name="visible">True</property>
                            <property name="can_focus">True</property>
                            <property name="border_width">3</property>
                            <property name="pixels_above_lines">1</property>
                            <property name="pixels_below_lines">1</property>
                            <property name="wrap_mode">word-char</property>
                            <property name="buffer">definitions_textbuffer</property>
                            <property name="accepts_tab">False</property>
                            <signal name="event-after" handler="gw_addvocabularywindow_definitions_event_after_cb" object="toplevel" after="yes" swapped="no"/>
                          </object>
                        </child>
                      </object>
                      <packing>
                        <property name="expand">True</property>
                        <property name="fill">True</property>
                        <property name="position">1</property>
                      </packing>
                    </child>
                  </object>
                  <packing>
                    <property name="expand">True</property>
                    <property name="fill">True</property>
                    <property name="position">3</property>
                  </packing>
                </child>
              </object>
              <packing>
                <property name="expand">True</property>
                <property name="fill">True</property>
                <property name="position">1</property>
              </packing>
            </child>
          </object>
          <packing>
            <property name="expand">True</property>
            <property name="fill">True</property>
            <property name="padding">8</property>
            <property name="position">1</property>
          </packing>
        </child>
      </object>
    </child>
    <action-widgets>
      <action-widget response="0">cancel_button</action-widget>
      <action-widget response="0">add_button</action-widget>
    </action-widgets>
  </object>
  <object class="GtkSizeGroup" id="entry_sizegroup">
    <property name="mode">both</property>
    <widgets>
      <widget name="vocabulary_list_combobox"/>
      <widget name="kanji_entry"/>
      <widget name="furigana_entry"/>
    </widgets>
  </object>
  <object class="GtkSizeGroup" id="label_sizegroup">
    <property name="mode">both</property>
    <widgets>
      <widget name="kanji_label"/>
      <widget name="furigana_label"/>
      <widget name="definitions_label"/>
      <widget name="list_label"/>
    </widgets>
  </object>
</interface>
